packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -20,15 +20,24 @@ function printInspectUrl(
   output: Output,
   deploymentUrl: string,
   deployStamp: () => number,
-  orgName: string
+  orgSlug: string
 ) {
-  const urlParts = deploymentUrl
-    .replace(/\..*/, '')
-    .replace('https://', '')
-    .split('-');
-  const deploymentShortId = urlParts.pop();
-  const projectName = urlParts.join('-');
-  const inspectUrl = `https://zeit.co/${orgName}/${projectName}/${deploymentShortId}`;
+  const url = deploymentUrl.replace('https://', '');
+
+  // example urls:
+  // lucim-fyulaijvg.now.sh
+  // s-66p6vb23x.n8.io (custom domain suffix)
+  const [sub, ...p] = url.split('.');
```

Comment:
You should use the `psl` module to parse this (`now.sh` is even [whitelisted](https://github.com/lupomontero/psl/blob/9775982e06068afbefad9dfb9d025b9d84bf1a9e/data/rules.json#L8805) as a public suffix):

```js
> psl.parse('asdfasdf.now.sh').tld
'now.sh'
```

Reply:
That's great! I don't think this makes it easier to detect custom suffixes though 🤔

Reply:
On second thought, the current code is more correct than what we can achieve with `psl`. I'd prefer to keep it as is.

Reply:
I don't think we support subdomains for custom suffix, but ok, the current code LGTM anyways.